src/App.tsx

```diff
@@ -9,10 +9,13 @@ import TableCell from '@mui/material/TableCell';
 import TableContainer from '@mui/material/TableContainer';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
+import { TableHead } from "@mui/material";
 
 
 function App() {
-
+  const [time, setTime] = useState<string[]>([]);
+  const [source, setSource] = useState<string[]>([]);
+  const [debug, setDebug] = useState<string[]>([]);
```

Comment:
Given that these three are only ever used together, I think they should be a single piece of state. Perhaps you need to make a class that holds all three?

Reply:
added to #19